Lib/test/test_tools/i18n_data/messages.py

```diff
@@ -32,23 +33,22 @@
 _(None)
 _(1)
 _(False)
-_(("invalid"))
```

Comment:
Perhaps this was intended to be a one-tuple? The following elements are a single-element list and set.

```suggestion
_(("invalid",))
```

Reply:
No, it tested a literal string nested in parentheses. Now this is supported.